multinet/auth/util.py

```diff
@@ -14,6 +17,7 @@
 def require_login(f: Any) -> Any:
     """Decorate an API endpoint to check for a logged in user."""
 
+    @functools.wraps(f)
```

Comment:
Is this just done to allow access to the underlying function?

Reply:
I think I explained it in a commit message, but without this decorator, the name of the returned function becomes the local name of the decorator function, which causes trouble for Flask--it thinks you're defining multiple routes named "wrapped" (or whatever the local decorator function name is).